superset-frontend/src/SqlLab/components/ColumnElement/index.tsx

```diff
@@ -80,7 +75,7 @@ interface ColumnElementProps {
   };
 }
 
-export default function ColumnElement({ column }: ColumnElementProps) {
+const ColumnElement: FC<ColumnElementProps> = ({ column }) => {
```

Comment:
I've seen instances where `React.FC` can be problematic ([example](https://github.com/typescript-cheatsheets/react#function-components)) - could you either define this more explicitly or write some tests to ensure this won't cause any issues?